pkg/controller/jobframework/reconciler.go

```diff
@@ -621,7 +620,18 @@ func (r *JobReconciler) getAncestorJobManagedByKueue(ctx context.Context, jobObj
 		if QueueNameForObject(parentJob) != "" {
 			return parentJob, nil
 		}
-		jobObj = parentJob
+		currentJob = parentJob
+		if len(seen) > managedOwnersChainLimit {
+			r.record.Eventf(jobObj, corev1.EventTypeWarning, ReasonJobNestingTooDeep,
+				"Terminated search for Kueue-managed Job because ancestor depth exceeded limit of %d", managedOwnersChainLimit)
+			ctrl.LoggerFrom(ctx).V(2).Info(
+				"Terminated search for Kueue-managed Job because ancestor depth exceeded managedOwnersChainlimit",
+				"jobObj", jobObj,
```

Comment:
```suggestion
				"job", klog.KObj(jobObj),
				"gvk", jobObj.GetObjectKind().GroupVersionKind(),
```

This can not be parsable, I guess.

Reply:
```suggestion
```

I remembered that we already store those infos to logger: https://github.com/kubernetes-sigs/kueue/blob/a1351a1866f88ba6814d3a7462cbbdf859ee1675/pkg/controller/jobframework/reconciler.go#L235

So, I think that we can just remove the info here.